app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainViewModel.kt

```diff
@@ -10,10 +10,10 @@ import edu.uc.coffeens.stocktracker.dto.Stock
 import edu.uc.coffeens.stocktracker.services.StockService
 
 class MainViewModel : ViewModel() {
-    var stock: MutableLiveData<ArrayList<Stock>> = MutableLiveData<ArrayList<Stock>>()
+    var stocks: MutableLiveData<ArrayList<Stock>> = MutableLiveData<ArrayList<Stock>>()
     var stockService: StockService = StockService()
     private lateinit var firestore : FirebaseFirestore
-    private var storageReferenence = FirebaseStorage.getInstance().getReference()
+
```

Comment:
The storageReference variable wasn't being used, so I removed it from the code to declutter. This was suggested during lectures and this article : https://medium.com/@dmstocking/kotlin-considered-harmful-unused-parameters-211846add4c0.